### DISCUSSION TITLE
Rendering Custom SVG instead of Inbuilt Chevrons

### DISCUSSION THREAD

**Discussion by JamCodesSometimes** [gpbl/react-day-picker#2482] [Support]
I'm just starting out with working on react-day-picker and want to change the chevrons to my custom svgs which is: 

<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"><path d="M12 2a10 10 0 1 0 10 10A10.011 10.011 0 0 0 12 2zm0 18a8 8 0 1 1 8-8 8.009 8.009 0 0 1-8 8z"/><path d="M13.293 7.293 8.586 12l4.707 4.707 1.414-1.414L11.414 12l3.293-3.293-1.414-1.414z"/></svg>

instead of the inbuilt Chevrons, but I'm unable to do so using the components prop.  I tried to follow another discussion about the Chevrons related to the rotation but still haven't found success. 

I would appreciate any help. I've not applied personal styling or customization yet.

**Comment by owieth** [comment]
You could do something like the following:

```
components={{
  Chevron: ({ orientation }) =>
    orientation === "left" ? (
      <ChevronLeft className="h-4 w-4" />
    ) : (
      <ChevronRight className="h-4 w-4" />
    ),
}}
```
